src/test/resources/log4j2.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I have the following issues with this conifg:
- we don't need "SSS" (milliseconds) in our pattern
- no context given
- default log level should be info

the log4j2.xml from the web-fixture looks good: https://github.com/test-editor/web-fixture/blob/develop/src/test/resources/log4j2.xml